src/modules.py

```diff
@@ -404,3 +404,13 @@ def parse_formbook(config: Dict[str, Any]) -> IocCollection:
         del config["c2_url"]
 
     return iocs
+
+
+@module("cobaltstrike")
+def parse_cobaltstrike(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+
+    for url in config.get("stager_url", []):
+        iocs.try_add_url(url["url"])
+
```

Comment:
We know for a fact that `config["urls"]` are broken (no schema):

![image](https://github.com/CERT-Polska/mwdb_iocextract/assets/7026881/a731ac4b-ce82-441a-ae74-69d266d70cf2)

But we know for a fact what's the protocol. What do you think about fixing them here (prepending schema depending on beacon_type)?

Anyway looks good. Add a test maybe?